tests/base/src/test/java/org/wildfly/security/ssl/SSLAuthenticationTest.java

```diff
@@ -17,12 +17,7 @@
  */
 package org.wildfly.security.ssl;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.fail;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*;
```

Comment:
Hi there! Could we change this wildcard import to more specific imports? I think the style checker will complain otherwise.